scenario/GovernanceScenario.ts

```diff
@@ -3,7 +3,7 @@ import { expect } from 'chai';
 import { constants, utils } from 'ethers';
 import { CometModifiedFactory, CometModifiedFactory__factory } from '../build/types';
 
-scenario('upgrade Comet implementation and initialize', { upgradeAll: true }, async ({ comet, configurator, proxyAdmin }, world, context) => {
+scenario.skip('upgrade Comet implementation and initialize', { upgradeAll: true }, async ({ comet, configurator, proxyAdmin }, world, context) => {
```

Comment:
So you are skipping these bc they actually need gov? Want to `XXX` these to mark and so we know to expect the coming fix?

Reply:
Yep. Sure, added the XXX.